modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/opt/GridH2Table.java

```diff
@@ -210,6 +214,56 @@ public GridH2Table(CreateTableData createTblData, GridH2RowDescriptor desc, H2Ro
         lock = new ReentrantReadWriteLock();
     }
 
+    /**
+     * @return Information about all indexes related to the table.
+     */
+    @SuppressWarnings("ZeroLengthArrayAllocation")
+    public List<IndexInformation> indexesInformation() {
+        List<IndexInformation> res = new ArrayList<>();
+
+        IndexColumn keyCol = indexColumn(KEY_COL, SortOrder.ASCENDING);
+
+        List<IndexColumn> wrappedKeyCols = H2Utils.treeIndexColumns(rowDescriptor(),
+            new ArrayList<>(2), keyCol, affKeyCol);
+
+        //explicit add HASH index, due to we know all their parameters and it doesn't created on non afinity nodes.
```

Comment:
there should be space after "//". Also fix line 237.

Reply:
typo in the word "affinity".

Reply:
fixed.